src/API/Products.php

```diff
@@ -112,12 +111,14 @@ protected function prepare_objects_query( $request ) {
 	 * @return WP_Error|WP_REST_Response
 	 */
 	public function get_items( $request ) {
-		add_filter( 'posts_fields', array( __CLASS__, 'add_wp_query_fields' ), 10, 2 );
+		if ( true === $request->get_param( 'low_in_stock' ) ) {
+			return ( new ProductsLowInStock() )->get_items( $request );
+		}
+
```

Comment:
Sorry @moon0326, I think I've had a change of mind about this after testing today.

I think it would be better to keep the old endpoint exactly as it was for the time being. At the moment we don't know what else is using the old endpoint. I just worry there might be some combination of parameters being passed to it that we haven't replicated in the new endpoint yet. I'd hate to mess up a use-case in the mobile app for example.

I think I'd like to learn about this further before we make changes to the old endpoint. We could do it within a major release where there is more time to make sure we've got everything right.

Reply:
Sounds good to me. I'll revert the changes (controller + test) 👍

Reply:
updated in https://github.com/woocommerce/woocommerce-admin/pull/7377/commits/b0a21fe087746e4317bce2c0c9b23e3f1cb9aa61 https://github.com/woocommerce/woocommerce-admin/pull/7377/commits/8829d3426d8eaba682b9d8abd299b0bf31f58d8f

Reply:
Much thanks !